### ISSUE TITLE
Is there any way to include file suffixes on files copied to the server

### ISSUE THREAD

**Issue by NotBrianZach** [krebs/krops#1]
seems like it would be nice to have, might think about it myself if it's not possible currently

**Comment by 4z3** [comment]
Can you rephrase that, I'm not exactly sure what's meant by "including file suffixes on files copied". To me it reads like "is there a way to just copy files with a given suffix". If that's the question, then it could be accomplished using rsync filters (`.rsync-filter` files). The rsync contains information about that.

**Comment by NotBrianZach** [comment]
https://github.com/krebs/krops/blob/master/pkgs/populate/default.nix
 looking in this file, there doesn't seem to be a way to include file suffix in the target path
unless something like this is possible

lib.evalSource [
"myFile.sql".file = toString /homedir/example.sql;
];
or I guess you could get the file suffix by sticking the file in a directory then using .file to include the directory

**Comment by 4z3** [comment]
> "myFile.sql".file = toString /homedir/example.sql;

This is possible.
